packages/api/endpoints/granule-csv.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const { Parser } = require('json2csv');
+const router = require('express-promise-router')();
+
+const { Granule } = require('../models');
+
+/**
+ * Builds a CSV file of all granules in the Cumulus DB
+ *
+ * @param {Object} req - express request object
+ * @param {Object} res - express response object
+ * @returns {Object} the csv file of granules
+ */
+async function list(req, res) {
+  const granuleModel = new Granule();
+  const allDbGranules = await granuleModel.scan();
+  const fields = ['granuleUr', 'collectionId', 'startDateTime', 'endDateTime'];
+  const granuleArray = [];
+  allDbGranules.Items.forEach((granule) => {
```

Comment:
Up to you, but you could replace the
```
const granuleArray = [];
forEach(() => {
  ...
  granuleArray.push
```
construction with the `map()` function:
```
const granuleArray = allDbGranules.Items.map((granule) => ...
```

Reply:
updated